ibm/service/kubernetes/resource_ibm_container_vpc_worker.go

```diff
@@ -635,3 +689,23 @@ func vpcClusterVpcWorkersVersionRefreshFunc(client v2.Workers, workerID, cluster
 		return worker, versionUpdating, nil
 	}
 }
+
+func validateSoftwareDefinedStorage(sds string) error {
+	var sdsList []string = []string{"ODF", "PTX"}
```

Comment:
Remove PTX from here and create a global/constant variable to save these strings.